seed/csharp-model/api-wide-base-path/.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: ci
+
+on: [push]
+
+jobs:
+  compile:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - uses: actions/checkout@master
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 7.0.404
+
+      - name: Install tools
+        run: |
+          dotnet tool restore
+
+      - name: Build Release
+        run: dotnet build src -c Release /p:ContinuousIntegrationBuild=true
+
+
+  publish:
+    needs: [compile]
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 7.0.404
+
+      - name: Publish
+        env:
+          NUGET_API_KEY: ${{ secrets. }}
```

Comment:
we make the key here `NUGET_API_KEY` and then reference `secrets.NUGET_API_TOKEN` in the actual command

Reply:
Updated!